ios/RNPdftron.m

```diff
@@ -109,6 +109,31 @@ - (void)setPassword:(NSString *)password onPDFDoc:(PTPDFDoc *)pdfDoc
     }
 }
 
+RCT_EXPORT_METHOD(pdfFromOfficeTemplate:(NSString *)docxPath json:(NSDictionary *)json resolver:(RCTPromiseResolveBlock)resolve rejecter:(RCTPromiseRejectBlock)reject)
+{
+    @try {
+        PTPDFDoc* pdfDoc = [[PTPDFDoc alloc] init];
+        NSData *jsonData = [NSJSONSerialization dataWithJSONObject:json options:0 error:nil];
+        NSString *jsonString = [[NSString alloc] initWithData:jsonData encoding:NSUTF8StringEncoding];
+
+        PTOfficeToPDFOptions* options = [[PTOfficeToPDFOptions alloc] init];
+        [options SetTemplateParamsJson:jsonString];
+
+        [PTConvert OfficeToPDF:pdfDoc in_filename:docxPath options:options];
+        NSString* homeDirectory = NSHomeDirectory();
+        NSString* resultPdfPath = [NSString stringWithFormat:@"%@%@", homeDirectory, @"/tmp.pdf"];
```

Comment:
Instead of calling every file tmp.pdf, I'd recommend giving every file a unique name, which the caller can then delete if they want to. You can create a random unique string with
`fileName = [NSUUID UUID].UUIDString;`